public/app/plugins/datasource/azuremonitor/components/ConfigEditor.tsx

```diff
@@ -110,7 +110,7 @@ export class ConfigEditor extends PureComponent<Props, State> {
             {error.details && <details style={{ whiteSpace: 'pre-wrap' }}>{error.details}</details>}
           </Alert>
         )}
-        {config.secureSocksDSProxyEnabled && (
+        {config.secureSocksDSProxyEnabled && ( // cant remove?
```

Comment:
This is the only place that still has a reference to Grafana core. Do you know if we still need this config option?

Reply:
It is still disabled by default, and it actually will continue to be disabled by default on OSS. It'll only be enabled by default in cloud

Reply:
I made a separate issue for this. Thank you @stephaniehingtgen https://github.com/grafana/grafana/issues/75966